### ISSUE TITLE
`dns_test` is never executed by the runner, and also fails

### ISSUE THREAD

**Issue by hakuch** [scylladb/seastar#521]
## Environment

Commit: c1e0e5d6b461bdd314d2b190dc9e31ccd92fd6d1
Platform: Fedora 28

## Details

The test corresponding to `tests/dns_test.cc` is never executed by `test.py`.

Furthermore, the test itself fails on my machine:

```
$ build/release/tests/dns_test 
Running 6 test cases...
WARN  2018-10-26 17:47:11,121 [shard 0] seastar - Unable to set SCHED_FIFO scheduling policy for timer thread; latency impact possible. Try adding CAP_SYS_NICE
unknown location(0): fatal error: in "test_resolve_tcp": std::system_error: Timeout
tests/test-utils.cc(40): last checkpoint

*** 1 failure is detected in the test module "tests/dns_test.cc"
```

The test should either be corrected and run regularly, or deleted.

**Comment by elcallio** [comment]
Output stream behaviour changed since dns support was written, tcp sends never flushed. Patch sent for this. Note however that the test itself can still fail if the resolved dns host does not support tcp query.

**Comment by espindola** [comment]
I just noticed the test still has

// Currently failing, disable until fixed (#521)
#if 0

Have we marked this as fixed too soon?
